cmd/operator-sdk/printdeps/cmd.go

```diff
@@ -51,39 +48,50 @@ func printDepsFunc(cmd *cobra.Command, args []string) error {
 	if len(args) != 0 {
 		return fmt.Errorf("command %s doesn't accept any arguments", cmd.CommandPath())
 	}
+	projutil.MustInProjectRoot()
```

Comment:
Should we support running this command from an arbitrary directory if `--dep-manager` is set? If unset outside of the project root, we'd return the `ErrNoDepManager` error. Would that be sufficient?

Reply:
The command still needs to infer the project type from project files to print the correct file version.

Reply:
Oh right, Go vs. Ansible vs. Helm. :+1: